utility/src/main/java/org/oppia/util/parser/HtmlParserEntityTypeModule.kt

```diff
@@ -21,6 +21,6 @@ class HtmlParserEntityTypeModule {
   @Provides
   @RevisionCardHtmlParserEntityType
   fun provideReviewCardHtmlParserEntityType(): String {
-    return "subtopic"
+    return "topic"
```

Comment:
@anandwana001 If you check this PR https://github.com/oppia/oppia-android/pull/1472 , You will notice in the description we can show subtopic rich-text images. Unfortunately I cannot merge that PR in develop because of content issues.
But you can see that I have made this change of `subtopic` to `topic` which is similar to what I had done on that PR in last file.